test/Google.Api.Gax.Tests/CallSettingsTest.cs

```diff
@@ -56,5 +58,60 @@ public void CloneNullHeaders()
             var clone = callSettings.Clone();
             Assert.Null(clone.Headers);
         }
+
+        [Fact]
+        public void ToCallOptions_ExpirationNull()
+        {
+            var mockClock = new Mock<IClock>();
+            CallSettings callSettings = new CallSettings();
+            var options = callSettings.ToCallOptions(mockClock.Object);
+            Assert.Null(options.Deadline);
+            mockClock.Verify(c => c.GetCurrentDateTimeUtc(), Times.Never);
+        }
+
+        [Fact]
+        public void ToCallOptions_ExpirationTimeout()
+        {
+            var now = new DateTime(2015, 6, 19, 5, 2, 3, DateTimeKind.Utc);
```

Comment:
FakeClock is designed to make this sort of thing easier, unless you *really* care how often the clock is being consulted. (I don't, personally...)